README.md

```diff
@@ -184,6 +184,29 @@ what's supported here: https://node.green/ (The reason is that our npm library
 must support earlier versions of Node, so the tests need to run on those
 versions.)
 
+### To use as a module in a Node.js project
```

Comment:
@BatDroid maybe the title should be "How to use with React-Native" -- since you can already use this library as a odule in Node.js

Reply:
oh i totally forgot about it. sorry my bad